src/intrinsics/ecma262/Error.js

```diff
@@ -74,10 +74,10 @@ function buildStack(realm: Realm, context: ObjectValue) {
     msg.throwIfNotConcrete();
   }
 
-  for (let executionContext of stack) {
+  for (let executionContext of stack.reverse()) {
     let caller = executionContext.caller;
     let locString = describeLocation(realm, caller ? caller.function : undefined, caller ? caller.lexicalEnvironment : undefined, executionContext.loc);
-    if (locString !== undefined) lines.unshift(locString);
+    if (locString !== undefined) lines.push(locString);
```

Comment:
I've also run a minor benchmark and my assumptions seem to hold - considerably faster to push vs. unshift